cmd/config/internal/commands/run-fns.go

```diff
@@ -111,106 +112,53 @@ func (r *RunFnRunner) runE(c *cobra.Command, args []string) error {
 
 // getContainerFunctions parses the commandline flags and arguments into explicit
 // Functions to run.
-func (r *RunFnRunner) getContainerFunctions(c *cobra.Command, dataItems []string) (
+func (r *RunFnRunner) getContainerFunctions(dataItems []string) (
```

Comment:
The linter had a bunch of complaints about this function, including that its cyclometric complexity was way too high. The changes here are mostly just function extractions, though there are also some boring linter changes in there that the extractions hide.